Plugins/Flow.Launcher.Plugin.Shell/Main.cs

```diff
@@ -203,7 +203,7 @@ private List<Result> ResultsFromHistory()
                 case Shell.Cmd:
                 {
                     info.FileName = "cmd.exe";
-                    info.Arguments = $"{(_settings.LeaveShellOpen ? "/k" : "/c")} {command} {(_settings.CloseShellAfterPress ? "& pause" : "")}";
+                    info.Arguments = $"{(_settings.LeaveShellOpen ? "/k" : "/c")} {command} {(_settings.CloseShellAfterPress ? $"&& echo {context.API.GetTranslation("flowlauncher_plugin_cmd_press_any_key_to_close")} && pause > nul /c" : "")}";
```

Comment:
iirc pause command already prompts press any key to continue or something similar. Why do we need another prompt?

Reply:
I thought it would be nicer to display another more fitting message, because `pause` only prompts `Press any key...` 
Also, for the PowerShell another message is needed anyway, because here `pause` prompts `Press the Enter key to continue...:` (but we want that any key can be pressed) and simply to have the same behavior I used the additional message for both

Reply:
Then we can just keep it for PowerShell. No need to `pause` in cmd.

Reply:
Sorry, I dont understand. Why is `pause` not needed in cmd?
So only: `info.Arguments = $"{(_settings.LeaveShellOpen ? "/k" : "/c")} {command} {(_settings.CloseShellAfterPress ? $"&& echo {context.API.GetTranslation("flowlauncher_plugin_cmd_press_any_key_to_close")} /c" : "")}";` ? But then the terminal does not remain open

Reply:
@VictoriousRaptor could you please explain your comment in more detail so that I can continue working on the PR